CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This is a required update for Mainnet users containing the configuration for the
 
 ### Bugs fixed
 - Ensure that Web3Signer stops the http server when a sigterm is received
+- Update postgresql to fix CVE-2024-1597
```

Comment:
This release has already gone out, I probably should have rotated the changelog :)

Reply:
fixed